tests/migration_test.go

```diff
@@ -336,7 +336,87 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 		}
 	}
 
+	setMigrationBandwidthLimitation := func(migrationBandwidth resource.Quantity) {
+		cfg := getCurrentKv()
+		cfg.MigrationConfiguration.BandwidthPerMigration = &migrationBandwidth
+		tests.UpdateKubeVirtConfigValueAndWait(cfg)
+	}
+
 	Describe("Starting a VirtualMachineInstance ", func() {
+
+		var pvName string
+		BeforeEach(func() {
+			pvName = "test-nfs" + rand.String(48)
+		})
+
+		const guestAgentMigrationTestName = " should be migrated successfully, using guest agent on VM "
```

Comment:
I am not sure this has some positive impact.

Reply:
What do you mean by that?

Reply:
The description of the test `It` is now `[test_id:2653]"+guestAgentMigrationTestName+"with default migration configuration`. I have to look up the variable if I want to know what is the name . Usually one would see that `...should be migrated successfully, using guest agent on VM with default migration...` test failed and went to the codebase to search for it(Searching by exact string would fail in this case). 
Last thing is that I don't think this is duplication nor this helps to prevent duplication.

Reply:
Yes, you have a valid point for sure. I'll change that.